### PR TITLE
taxonomy: around 60 ids from Ecobalyse have been added for few ingredients

### PR DESCRIPTION
**Pull request by very-smartin**
### What

In the ongoing process of the manual mapping between Ecobalyse and OFF, around 60 ids from Ecobalyse have been added for few ingredients.

### Related issue(s) and discussion

- Fixes #[ISSUE NUMBER]

### CODE REVIEW

File: taxonomies/food/ingredients.txt [hunk 1]
```diff
@@ -44020,6 +44115,9 @@ zh: 樱
 # openfoodfacts:https://world.openfoodfacts.org/ingredient/cherry
 carbon_footprint_fr_foodges_ingredient:fr: Cerise
 carbon_footprint_fr_foodges_value:fr: 1.3
+ecobalyse:en: cherry-tr
```
Comment (stephanegigandet) [inline taxonomies/food/ingredients.txt:44118]: cherry-fr ?

File: taxonomies/food/ingredients.txt [hunk 2]
```diff
@@ -17252,16 +17282,21 @@ en: refined rapeseed oil
 hr: rafinirano ulje repice
 lt: rafinuotas rapsų aliejus
 pl: rafinowany olej rzepakowy, olej rzepakowy rafinowany
+ecobalyse:en: refined-rapeseed-oil
 
 < en:rapeseed oil
 en: british rapeseed oil
 de: Britisches Rapsöl
 hr: britanskog repičinog ulja
+ecobalyse:en: rapeseed-oil
```
Comment (stephanegigandet) [inline taxonomies/food/ingredients.txt:17291]: Can you leave only the ecobalyse properties for rapeseed oil on the parent en:rapeseed-oil and remove it from its children? (this one and more below)

Comment (very-smartin) [reply taxonomies/food/ingredients.txt:17291]: Noted ! It will be corrected in the next PR

**Review by stephanegigandet — APPROVED** [review]
Thank you!

**Pull request merged**